sdk/openai/openai/test/public/utils/utils.ts

```diff
@@ -0,0 +1,203 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { assert, TaskContext, Test, TestContext } from "vitest";
+import {
+  PipelineRequest,
+  PipelineResponse,
+  RestError,
+  createDefaultHttpClient,
+  createEmptyPipeline,
+  createHttpHeaders,
+  createPipelineRequest,
+} from "@azure/core-rest-pipeline";
+import { CognitiveServicesManagementClient } from "@azure/arm-cognitiveservices";
+import { createTestCredential } from "@azure-tools/test-credential";
+import { assertEnvironmentVariable } from "@azure-tools/test-recorder";
+import {
+  EnvironmentVariableNamesAzureCommon,
+  EnvironmentVariableNamesForAzureSearch,
+  EnvironmentVariableNamesForCompletions,
+  EnvironmentVariableNamesForDalle,
+  EnvironmentVariableNamesForWhisper,
+} from "./envVars.js";
+import { logger } from "@azure/identity";
```

Comment:
Is this a mistake?

Reply:
Yes, it doesn't have dev dependency on the logger package so I update it in `package.json` now